include/wx/setup_inc.h

```diff
@@ -757,7 +757,7 @@
 
 // notice that we can't use wxCHECK_VISUALC_VERSION() here as this file is
 // included from wx/platform.h before wxCHECK_VISUALC_VERSION() is defined
-#ifdef _MSC_VER
+#if defined(_MSC_VER) || defined(__MINGW32__)
```

Comment:
hi @vadz it seems I cannot use __MINGW64_TOOLCHAIN__ on wx/setup.h because the wx/platform.h using that file before defining the __MINGW64_TOOLCHAIN__. Do you have any suggestion!?